Increment deployment target from iOS 12 to 13

esreli opened the pull request:
This PR drops support for iOS 12, removes properties deprecated by iOS 13, and increments the deployment target to iOS 13.
[[Related Issue]](https://github.com/ArcGIS/open-source-apps/issues/458)

philium commented:
> [[Related Issue]](https://github.com/ArcGIS/open-source-apps/issues/458)

That link seems to be broken.

philium commented:
Does the README need to be updated?

philium commented:
The deployment target of the two test targets should be updated too.

esreli commented:
> That link seems to be broken.

I've added you to the github `ArcGIS/open-source-apps` team. The link should now be accessible.

mhdostal commented:
> Does the README need to be updated?

We don't list iOS version information in the README, but adding a line to the Release notes might be a good idea, since we are ahead of the SDK a bit.

mhdostal reviewed (approved):
Approved, unless we want to add something to the RELEASE notes.

esreli commented:
> Does the README need to be updated?

At first glance, I didn't think I needed to. Upon further reflection (nudged by your suggestion), there are two areas where the README now reflects the iOS 13 deployment target.

1. Note on simulating the app on macOS Mojave
1. Requirements- requires a device or simulator running/simulating iOS 13.0

> The deployment target of the two test targets should be updated too.

They're now updated.

> unless we want to add something to the RELEASE notes.

I've updated the release notes.

@mhdostal @philium would you re-review?
